packages/react-icons/convert-font.js

```diff
@@ -154,12 +177,14 @@ async function generateCodepointMapForWebpackPlugin(destPath, iconEntries, resiz
  *
  * @param {Record<string, number>} iconEntries
  * @param {boolean} resizable
- * @returns {string[]}
+ * @returns {{ entries: string[], names: string[] }}
  */
 function generateReactIconEntries(iconEntries, resizable) {
   /** @type {string[]} */
   const iconExports = [];
-  const metadata = JSON.parse(fsS.readFileSync(RTL_FILE, 'utf-8'));
+  /** @type {string[]} */
+  const iconNames = [];
+  const metadata = JSON.parse(fsS.readFileSync(/** @type {string} */ (RTL_FILE), 'utf-8'));
   for (const [iconName, codepoint] of Object.entries(iconEntries)) {
     let destFilename = getReactIconNameFromGlyphName(iconName, resizable);
     var flipInRtl = metadata[destFilename] === 'mirror';
```

Comment:
nit, and might be out of scope for the PR, but:

```suggestion
    const flipInRtl = metadata[destFilename] === 'mirror';
```

Reply:
yeah I plan to clean things up after "in review" prs land in a batch
- adding prettier
- modernizing the scripts etc